nixos/modules/services/web-servers/varnish/default.nix

```diff
@@ -7,32 +7,48 @@ with lib;
 {
   options = {
     services.varnish = {
-      enable = mkOption {
-        default = false;
-        description = "
-          Enable the Varnish Server.
-        ";
-      };
+      enable = mkEnableOption "Varnish Server";
 
       http_address = mkOption {
```

Comment:
I'd also changed it to ```httpAddress``` for style reasons but it may break someone's setup (or may be not? considering the module was broken many months, it should have no users at all)